pkg/router/controller/host_admitter.go

```diff
@@ -165,14 +152,22 @@ func (p *HostAdmitter) SetLastSyncProcessed(processed bool) error {
 	return p.plugin.SetLastSyncProcessed(processed)
 }
 
+// recordRejection records why the route was rejected.
+func (p *HostAdmitter) recordRejection(route *routeapi.Route, reason, message string) {
+	oldPolicy := route.Spec.WildcardPolicy
+	route.Spec.WildcardPolicy = routeapi.WildcardPolicyNone
```

Comment:
Its to set it on the status/RouteIngress field - its a lot of change to the other plugins if we want to pass it down as a parameter.